overview/content-types/commands.nix

```diff
@@ -46,20 +46,18 @@ in
         if lib.isList self.instructions then
           ''
             <ul>
-              ${lib.concatMapStringsSep "\n" (i: ''
-                <li>
-                  <dt>${i.platform}</dt>
-                  <dd>
-                    ${i.commands.bash}
-                  </dd>
-                </li>
-              '') self.instructions}
+            ${lib.concatMapStringsSep "\n" (i: ''
+              <li>
+              <details>
```

Comment:
I wouldn't go into these skinning details here though and instead focus on the mechanism. My plan is to use a radio button and control visibility via CSS, because you want a single-select behavior. Details will require closing a collapsed section to get it out of the way, that's way too much interaction.